public/content/documentation/web/component/breadcrumbs.md

```diff
@@ -0,0 +1,104 @@
+## General Notes
+
+How to test a breadcrumbs
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+1. Keyboard & screen reader actions
+   - Tab: Focus visibly moves to the link
+   - Enter: Activates the link
+
+2. Mobile screenreader gestures
+   - Swipe: Focus moves to the link
+   - Doubletap: Activates the link
+
+3. Screenreader output for all devices
+   - Name: The link names correspond to their destination page titles
+   - Role: Links identify as a links in a breadcrumb navigation landmark
+   - State: The current page link is indicated when focused
+   - Group: Is discoverable with screenreader shortcuts as a navigation landmark
+
+Full information: [https://www.magentaa11y.com/checklist-web/breadcrumbs/](https://www.magentaa11y.com/checklist-web/breadcrumbs/)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a breadcrumbs
+
+GIVEN THAT I am on a page with a breadcrumbs
+
+1. Keyboard for mobile & desktop
+   - WHEN I use the tab key to move focus to a link
+   - I SEE focus is strongly visually indicated
+   - THEN when I use the enter key to activate the link
+   - I SEE my browser goes somewhere
+
+2. Desktop screenreader
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND I use the tab key to move focus to a link
+   - I HEAR The link names correspond to their destination page titles
```

Comment:
I think these "I HEAR" ones are indented again. Check out the button gherkin to see what I mean.